src/pages/identity/Identity.js

```diff
@@ -1,10 +1,47 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { connectIdmWallet } from 'react-idm-wallet';
+import styles from './Identity.css';
+import IdentityDetails from './IdentityDetails';
+import IdentityActivity from './IdentityActivity';
+import PageLayout from '../../shared/components/page-layout/PageLayout';
+import backgroundPatternUrl from '../../shared/media/backgrounds/background-pattern-2160p.png';
 
-const Identity = ({ match }) => <div>This is the identity page of { match.params.id }!</div>;
+class Identity extends Component {
+    render() {
+        const { id } = this.props.match.params;
+
+        if (!this.props.isLoaded()) {
+            return null;
+        }
+
+        return (
+            <div className={ styles.splitWrapper }>
+                <div className={ styles.split }>
+                    <div className={ styles.background } style={ { backgroundImage: `url(${backgroundPatternUrl})` } } />
+                    <PageLayout>
+                        <div className={ styles.top }>
+                            <IdentityDetails matchId={ id } />
```

Comment:
It feels strange for this property to be called `matchId`. Why not `id` or `identityId`? Change this everywhere.